features-json/same-site-cookie-attribute.json

```diff
@@ -274,7 +274,8 @@
       "10.0-10.2":"n",
       "10.3":"n",
       "11.0-11.2":"n",
-      "11.3":"n"
+      "11.3":"n",
+      "12": "y"
```

Comment:
I added Safari 12 for iOS here, not sure if that was ok?

Reply:
Its not okay. Please remove it.